### DISCUSSION TITLE
ETA on rollout of --esm option?

### DISCUSSION THREAD

**Discussion by TonyLB** [ohmjs/ohm#361] [General]
Hullo!

I'm loving using Ohm, and looking with barely restrained craving at the `--esm` option that I see in the source-code for the Ohm-JS/CLI module in 16.3.0.  It's not a _huge_ thing, but with AWS Lambda recently supporting ESM import syntax, I could keep things simpler if I could purge this last little big of CommonJS from my packages.

So I'm mostly weighing in to encourage you and offer my gratitude for the amazing toolkit you've offered here ... and maybe to roll my (admittedly weak) puppy-dog eyes and ask if you happen to know when the CLI release will be updated to serve out the new code.

Thanks,
Tony

**Comment by pdubroy** [comment]
Thanks for the kind words, and glad you are enjoying Ohm!

And thanks for the reminder — I just published [@ohm-js/cli v1.1.0](https://www.npmjs.com/package/@ohm-js/cli/v/1.1.0). Sorry for not releasing it sooner!
